docs/getting-started.md

```diff
@@ -0,0 +1,170 @@
+        id 'org.parchmentmc.librarian.forgegradle' version '1.+'
+    }
+    ```
+
+3. **Update your mappings channel and version** to the `parchment` channel and the appropriate version string.
+
+    ```gradle
+    minecraft {
+        mappings channel: 'parchment', version: '2021.08.15-1.17.1'
+    }
+    ```
+
+    > For using Parchment on the same version of Minecraft:
+    > : <code class="version"><u>YYYY</u>.<u>MM</u>.<u>DD</u>-<u>Minecraft version</u></code>
+    >
+    > > _Examples:_
+    > >
+    > > - **`2021.12.12-1.17.1`** for Minecraft 1.17.1
+    > > - **`2022.08.07-1.18.2`** for Minecraft 1.18.2
+    > > - **`2022.08.14-1.19.2`** for Minecraft 1.19.2
+    > {:.note.icon.note-example}
+    >
+    > For using Parchment for an older version on a newer MC version
+    > : <code class="version"><u>Mapping's MC version</u>-<u>YYYY</u>.<u>MM</u>.<u>DD</u>-<u>Environment MC version</u></code>
+    >
+    > > _Examples:_
+    > >
+    > > - **`1.17.1-2021.12.12-1.18`**
+    > >   - Minecraft _1.17.1_ mappings (`2021.12.12-1.17.1`) in an MC _1.18_ environment
+    > > - **`1.18.2-2022.08.07-1.19.1`**
+    > >   - Minecraft _1.18.2_ mappings (`2021.08.07-1.18.2`) in an MC _1.19.1_ environment
+    > > - **`1.19.2-2022.08.14-1.20`**
+    > >   - Minecraft _1.19.2_ mappings (`2021.08.14-1.19.2`) in an MC _1.20_ environment
+    > {:.note.icon.note-example}
+    {:.note.yellow-note}
+
+{% endtab %}
+
+{% tab parchment-setup Fabric and Fabric Loom %}
+
+### Fabric and Fabric Loom
+
+[**Fabric Loom**](https://github.com/FabricMC/fabric-loom) provides native support for Parchment mappings as of version 0.9.
+
+1. **Install the Parchment repository** in your buildscript's `repositories` block.
+
+    ```gradle
+    repositories {
+        maven {
+            name = 'ParchmentMC'
+            url = 'https://maven.parchmentmc.org'
+        }
+    }
+    ```
+
+2. **Use the `loom.layered()` mappings dependency** with Parchment mappings and official/Mojang mappings.
+
+    ```gradle
+    dependencies {
+        // ...
+        mappings loom.layered() {
+            officialMojangMappings()
+            parchment("org.parchmentmc.data:parchment-1.17.1:2021.09.05@zip")
+        }
+    }
+    ```
+
+    > The dependency should be formatted as follows:
+    > <code class="version">org.parchmentmc.data:parchment-<u>Minecraft version</u>:<u>YYYY</u>.<u>MM</u>.<u>DD</u>@zip</code>
+    {:.note.yellow-note}
+
+3. _(Optional)_ Run the `genSources` task to (re)generate the Minecraft sources JAR.
+
+    If your IDE fails to auto-detect the sources JAR when browsing Minecraft classes, manually select the JAR file ending with `-sources.jar` when prompted by your IDE.
+
+{% endtab %}
+
+{% endtabs %}
+
+---
+
+**Thank you for choosing Parchment mappings!** For any questions, please visit [the official Discord server](https://discord.parchmentmc.org/).
+
+[1.17-branch]: https://github.com/ParchmentMC/Parchment/tree/versions/1.17.x
```

Comment:
whasis?

Reply:
This is used in the second paragraph as an example of looking at the README of the particular branch to see its latest release version, which I carried over from the wiki page.